lib/modules/apostrophe-images/lib/cursor.js

```diff
@@ -80,6 +84,10 @@ module.exports = {
       }
     });
 
+    // Filter. Get images depending of their
```

Comment:
"depending on their" typo

Reply:
We need to provide some examples of extensions to use. Lower case? Upper case? With the leading period? We should be very clear, though we don't need to provide every possible option.

Reply:
@abea  Same as above, the user has nothing to configure here, and options are dynamics from `self.fileGroups`.
Basically for now there are only images extensions like `jpg`, `png`, `gif` and `svg` if allowed in `apostrophe-attachments`.

Reply:
Documentation improved.